aws/src/main/java/org/apache/iceberg/aws/glue/GlueCatalog.java

```diff
@@ -286,8 +286,12 @@ protected String defaultWarehouseLocation(TableIdentifier tableIdentifier) {
   }
 
   @Override
-  public List<TableIdentifier> listTables(Namespace namespace) {
-    namespaceExists(namespace);
+  public List<TableIdentifier> listTables(Namespace namespace) throws NoSuchNamespaceException {
+    // check if namespace exists
```

Comment:
this comment is redundant and can be removed

Reply:
Removed the redundant comment.